xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+   
+   <listitem>
+    <para>
+     Start &slsa; instances on different compute nodes that can ping their gateway and each other.
+    </para>
+
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ 
+ <sect1 xml:id="sec.prereqisites">
+  <title>Prequisites</title>
+    <para>
+     As the Cisco ACI fabric is customized and highly-variable per customer's needs, you must have your ACI fabric already set up and working, and you need your own ACI expert administrator.
+    </para>
+    <para>
+    If you are using Cisco UCS B series hardware there are additional prerequisites; please contact Cisco for information.
+    </para>
+    <para>
+    Start with a newly-installed &cloud; with all nodes discovered and allocated, and no barclamps yet deployed. (If you are using an existing installation then pay extra attention to all configurations, and check for duplicates and conflicts.)
+    </para>
+    <para>
+    Make sure the control node has an IP address that allows it to communicate to your Cisco APIC.
+   </para>
+    <para>
+     The examples in this document reference a simplified example network with these nodes:
+ </para>
+ 
+  <itemizedlist>
+   <listitem>
+    <para>
+        Two Cisco ACI leafs.
+    </para>
+   </listitem> 
+   
+   <listitem>
+    <para>
+        One admin node.
+    </para>
+   </listitem> 
+   
+    <listitem>
+    <para>
+        One control node.
+    </para>
+   </listitem> 
+   
+   <listitem>
+    <para>
+        Two compute nodes.
+    </para>
+   </listitem>  
+</itemizedlist>
+
+    <para>
+   The example network uses 192.168.124.0/24 for the admin network. Figure J-1 shows the topology of the example network.
+    </para>
+    
+      <figure>
+     <title>The example network topology.</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata fileref="depl_opflex_1.png" width="100%" format="png"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="depl_opflex_1.png" width="75%" format="png"/>
+      </imageobject>
+     </mediaobject>
+    </figure>
+</sect1>
+
+    <sect1 xml:id="sec.networking">
+  <title>Configure Networking on Compute Nodes</title>     
+    <para>
+In the example network we have two compute nodes that are connected to our Cisco ACI fabric. Network configuration will be automated in future releases, but for now you must follow these steps to configure networking on all of your compute nodes.
+    </para>
+    
+<procedure>
+    <step>
+        <para>
+            Enter <literal>YAST | System | Network Settings</literal> (Click Continue when you see the Chef client warning).
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Highlight the Ethernet device connected to the ACI fabric and press F4 (Edit).
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+        Enable <literal>No Link and IP Setup</literal>.
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+       Go to the General tab and set the MTU to 1600.
```

Comment:
All UI elements need to be wrapped ion &lt;guimenu>, this also includes tab-titles. For values we use &lt;literal>:

    Go to the <guimenu>General</guimenu> tab and set the <guimenu>MTU</guimen> to <literal>1600</literal>.